dev-libs/optix/optix-7.0.0.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="NVIDIA Ray Tracing Engine"
+HOMEPAGE="https://developer.nvidia.com/optix"
+SRC_URI="NVIDIA-OptiX-SDK-${PV}-linux64.sh"
+S="${WORKDIR}"
+
+LICENSE="NVIDIA-SDK"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="bindist fetch"
+
+pkg_nofetch() {
+	einfo "Please download ${SRC_URI} from:"
+	einfo "  ${HOMEPAGE}"
+	einfo "and move it to your DISTDIR directory."
+}
+
+src_unpack() {
+	tail -n +223 "${DISTDIR}"/${A} | tar -zx
+	assert "unpacking ${A} failed"
+}
+
+src_install() {
+	insinto /usr/include/${PN}
+	doins -r include/.
+
+	dodoc -r doc/.
```

One more small thing, noticed doc/. includes licenses (normally don't install those) and also third party licenses for things we haven't installed given headers-only (kinda misleading).
```suggestion
	dodoc doc/OptiX_{API_Reference,Programming_Guide}_${PV}.pdf
```
If installed SDK stuff, it's something I was going to have a closer look at given we'd need more LICENSE, but not an issue now.

---

Have you checked whether the license permits us to do this? I don't see a big win by not installing files worth <500kB of disk space. 
Paragraph 1.1 (i) only allows to "[m]ake modifications and create derivative works of the source source code and header files[...]" of the "Licensed Software". The term "Licensed Software" includes documentation as per the first paragrah of the license. In 2.1 (v) removal of any proprietary notice that appears on or with the Licensed Software or any copies thereof is explicitly forbidden.

---

The license is already installed, you have it in this commit. It's a duplicate.

---

Also see https://bugs.gentoo.org/782841 if you want to one of the recent debate regarding this. I am not going to debate it myself, and just going to go with the current guidelines that licenses@ agree with.

---

My question was, if the license permits us to simply remove the 2 documents from the installation. From my understanding this is not allowed.

---

One document is not used as far as I can see because we don't install the sdk files. We don't need third party notice for glfw if we don't install glfw. If we did, then we'd list them in LICENSE and install any additional license files in licenses/

The other is already available in licenses/, as it wants. We're not skipping it.

So this is removing a duplicate, and an unused file.

---

Besides, this is RESTRICT=bindist and only exists on the user's system.

---

Just read the bug you mentioned. I'm not 100% convincend, but as per advice from licenses@, I think it's ok to not install the two files.